lib/camera_screen.dart

```diff
@@ -142,10 +137,9 @@ class _CameraScreenState extends State<CameraScreen> {
           return;
         }
 
-        await Navigator.of(context).push(
-          MaterialPageRoute(builder: (context) => DisplayPictureScreen(imagePath: image.path))
-        );
-      } 
+        await Navigator.of(context).push(MaterialPageRoute(
+            builder: (context) => DisplayPictureScreen(imagePath: image.path)));
+      }
```

Comment:
I think there's also a merge conflict here, make sure to update it with what is on the main branch, not this one.

Reply:
it should be:

builder: (context) => ImageEditScreen(imagePath: image.path))); 

instead of DisplayPictureScreen